install/install.sh

```diff
@@ -28,10 +28,13 @@ echo "* Updating package list"
 apt-get update >> "$install_log" 2>&1
 echo "* Installing nginx, python, pip, kvm, libvirt and virt-manager"
 apt-get -q -y install nginx python python-dev python3-libvirt libxslt1-dev libxml2-dev python-pip qemu-kvm libvirt-bin virt-manager mediainfo >> "$install_log" 2>&1
-if [ ! -f /etc/init.d/mysql* ]; then
-    echo "* Installing MySQL (root password will be empty!)"
-    DEBIAN_FRONTEND=noninteractive apt-get install -y mysql-server >> "$install_log" 2>&1
-fi
+for file in /etc/init.d/mysql*
```

Comment:
Will this loop run if there's no files at all that are matching this pattern?

Reply:
Yes, it works. I've tested for that :smile: